packages/form-core/src/choice-group/ChoiceGroupMixin.js

```diff
@@ -71,6 +71,10 @@ const ChoiceGroupMixinImplementation = superclass =>
       // user are relevant, choice values are always defined by the Application Developer
```

Comment:
Type issues above can be solved in https://github.com/ing-bank/lion/blob/master/packages/form-core/types/choice-group/ChoiceGroupMixinTypes.d.ts